src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -440,6 +440,23 @@ export default {
 			}
 		},
 
+		messageButtonsBarProps() {
+			return {
+				actorType: this.actorType,
+				actorId: this.actorId,
+				message: this.message,
+				messageParameters: this.messageParameters,
+				timestamp: this.timestamp,
+				id: this.id,
+				isReplyable: this.isReplyable,
+				token: this.token,
+				messageType: this.messageType,
+				previousMessageId: this.previousMessageId,
+				lastEditActorDisplayName: this.lastEditActorDisplayName,
+				lastEditTimestamp: this.lastEditTimestamp,
+			}
+		},
```

Comment:
If we already consider it an atomic object, then it makes sense to pass it as a single prop.

Reply:
Not really, there is `isReplyable` and it doesn't cover the whole message object that we commonly use.

Reply:
> there is `isReplyable`

Which is a part of message object. The only thing missing from it is `previousMessageId`, and its usage looks minor, so I'd pass it separately and think of a way to not use it in the component at all (by modifying store action?)

Reply:
If we consider some specific props to not be a part of the `message` object we can pass them separately, keeping all the other properties a part of the `message` object.

So, if we want to refactor it, let's in both `Message` and `MessageButtonsBar` add a prop for `message` object with all the properties from the original `message` object (from store = API) and keep only additional properties as separated props.

Reply:
Refactored